forms-flow-bpm/forms-flow-bpm-camunda/src/main/java/org/camunda/bpm/extension/hooks/adapter/mail/response/EmptyResponse.java

```diff
@@ -0,0 +1,32 @@
+/* Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Can you move this out of mail package, as this class would be useful for other connectors too